.circleci/config.yml

```diff
@@ -111,37 +116,39 @@ workflows:
       - lint:
           name: Lint
           context: nodejs-install
-          node_version: '14'
+          node_version: '16.20'
```

Comment:
Looks like CLI tests from v12, so lets add those back in https://github.com/snyk/cli/blob/295a14789a0b55d9f1ad71607b8655520566aa09/.github/workflows/smoke-tests.yml#L20

Reply:
```
#!/bin/bash -eo pipefail
npm test

> snyk-nuget-plugin@ test /root/snyk-nuget-plugin
> npm run lint && npm run unit-test


> snyk-nuget-plugin@ lint /root/snyk-nuget-plugin
> prettier --check "./lib/**/*.ts" && eslint -c .eslintrc.js "./lib/**/*"

prettier requires at least version 14 of Node, please upgrade
```

Reply:
Guess I can live with earlier versions of dev-deps for a while. But this should REALLY be fixed in CLI.